docs/mast/mast_catalog.rst

```diff
@@ -24,7 +24,7 @@ The returned fields vary by catalog, find the field documentation for specific c
 `here <https://mast.stsci.edu/api/v0/pages.html>`__.
 If no catalog is specified, the Hubble Source Catalog will be queried.
 
-.. doctest-remote-data::
+.. doctest-skip::
```

Comment:
why switch to skip here?

Reply:
This one was kind of weird, it was failing because the actual output does not match the expected. However, I can't reproduce the results from the tests. 

Whenever I run the code on its own, I get the output that's currently in the documentation.

Reply:
The new commit doesn't skip the test, but ignores the output of the offending line.